ansible_collections/arista/avd/roles/eos_designs/templates/designs/l3ls-evpn/tenants/vlan-interfaces.j2

```diff
@@ -49,8 +49,8 @@ vlan_interfaces:
     ip_address: {{ tenants[tenant].vrfs[vrf].svis[svi].nodes[inventory_hostname].ip_address }}
 {%             endif %}
 {# Virtual Router IP Address #}
-{%             if svi_ip_virtual_router_address is arista.avd.defined %}
-    ip_virtual_router_address: {{ svi_ip_virtual_router_address | ansible.netcommon.ipaddr('address') }}
+{%             if svi_ip_virtual_router_addresses is arista.avd.defined %}
+    ip_virtual_router_addresses: {{ svi_ip_virtual_router_addresses | ansible.netcommon.ipaddr('address') }}
```

Comment:
This filter removes the mask.

Reply:
I think we should remove this extra `ansible.netcommon.ipaddr('address')`filter so we can accept `< IPv4 Address>` _and_ `< IPv4 Address/Mask >`